dictionaries/en_US/src/acronyms.txt

```diff
@@ -16,5 +16,7 @@ scotus
 SCOTUS
 tldr
 TLDR
+usd
+USD
```

Comment:
Wow, I had no idea USD was missing from the dictionary.

Reply:
I know, that was unexpected.